packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -54,7 +69,15 @@ class AccountImportOptions extends Component {
     } = this.props;
 
     this.setState({ isLoading: true });
+
     try {
+      const json = JSON.parse(jsonString);
```

Comment:
Do not parse here.

3 lines below, the `await setJsonString(jsonString);` actually parses the JSON, and createAccountStore.address will hold the parsed address. Do the `hasExistingAddressForImport` then.